cpp/arcticdb/column_store/chunked_buffer.cpp

```diff
@@ -12,106 +14,132 @@
+      source_bytes -= this_write;
+      target_pos += this_write;
+      remaining_current_bytes -= this_write;
+      remaining_total_bytes -= this_write;
+      ARCTICDB_DEBUG(log::version(),
+                     "Adjusted values source_pos {} source_bytes {} target_pos {} "
+                     "remaining_current {} remaining_total {}",
+                     source_pos, source_bytes, target_pos, remaining_current_bytes,
+                     remaining_total_bytes);
 
-            if(static_cast<size_t>((*target_block)->bytes()) == nbytes || target_pos == static_cast<size_t>((*target_block)->bytes())) {
-                ARCTICDB_DEBUG(log::version(), "Incrementing block as nbytes == target block bytes: {}", nbytes);
-                ++target_block;
-                target_pos = 0;
-            }
+      if (static_cast<size_t>((*target_block)->bytes()) == nbytes ||
+          target_pos == static_cast<size_t>((*target_block)->bytes())) {
+        ARCTICDB_DEBUG(log::version(),
+                       "Incrementing block as nbytes == target block bytes: {}",
+                       nbytes);
+        ++target_block;
+        target_pos = 0;
+      }
 
-            if(remaining_current_bytes == 0) {
-                ARCTICDB_DEBUG(log::version(), "Pushing buffer");
-                output.push_back(std::move(*current_buf));
-                current_buf.reset();
-            }
-        }
+      if (remaining_current_bytes == 0) {
+        ARCTICDB_DEBUG(log::version(), "Pushing buffer");
+        output.push_back(std::move(*current_buf));
+        current_buf.reset();
+      }
     }
-    util::check(output.size() == output_size, "Unexpected size in chunked buffer split {} != {}", output.size(), output_size);
-    return output;
+  }
+  util::check(output.size() == output_size,
+              "Unexpected size in chunked buffer split {} != {}", output.size(),
+              output_size);
+  return output;
 }
 
-template std::vector<ChunkedBufferImpl<64>> split(const ChunkedBufferImpl<64>& input, size_t nbytes);
-template std::vector<ChunkedBufferImpl<3968>> split(const ChunkedBufferImpl<3968>& input, size_t nbytes);
+template std::vector<ChunkedBufferImpl<64>> split(const ChunkedBufferImpl<64>& input,
+                                                  size_t nbytes);
+template std::vector<ChunkedBufferImpl<3968>>
+split(const ChunkedBufferImpl<3968>& input, size_t nbytes);
 
 // Inclusive of start_byte, exclusive of end_byte
 template <size_t BlockSize>
-ChunkedBufferImpl<BlockSize> truncate(const ChunkedBufferImpl<BlockSize>& input, size_t start_byte, size_t end_byte) {
-    ARCTICDB_DEBUG(log::version(), "Truncating buffer of size {} between bytes {} and {}", input.bytes(), start_byte, end_byte);
-    const auto output_size = start_byte >= end_byte ? 0 : end_byte - start_byte;
-    if(input.num_blocks() == 0 || output_size == 0)
-        return {};
+ChunkedBufferImpl<BlockSize> truncate(const ChunkedBufferImpl<BlockSize>& input,
```

Comment:
The one thing that bugs me most about the LLVM code styling is how it formats function arguments when there are many of them. Lining it with the end of the function name is odd. If you change the function name you have to reformat all arguments. That being said I'm really afraid of this becomming some sort of code styling flame war. So I'll just get over it.

Reply:
No, I agree (commented below). We can set BreakBeforeParameters to always and it should fix it I think